.github/workflows/conformance.yml

```diff
@@ -0,0 +1,79 @@
+name: Manually run upstream conformance test
+
+on:
+  workflow_dispatch:
+    inputs:
+      antrea-version:
+        description: The Antrea version to test. It could be a SHA-1 value, a branch, or a tag (e.g. a7b012b, release-1.12, v1.12.0).
+        default: main
+        required: true
+      k8s-version:
+        description: K8s version (e.g. v1.27.1). Kind's default K8s version will be used if empty.
+        required: false
+      test-suite:
+        description: The test suite to run. Valid values include "whole-conformance", "conformance", "network-policy, "sig-network", and "all".
+        default: whole-conformance
+        required: true
+      always-upload-logs:
+        description: Always upload logs regardless of the test result.
+        type: boolean
+        default: false
+
+env:
+  KIND_VERSION: v0.18.0
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: [ubuntu-latest]
+    steps:
+      - name: Free disk space
+        # https://github.com/actions/virtual-environments/issues/709
+        run: |
+          sudo apt-get clean
+          df -h
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.antrea-version }}
+          fetch-depth: 0
+      - name: Check if it is a released version
+        id: check-release
+        run: |
+          if git show-ref --tags --verify --quiet refs/tags/${{ inputs.antrea-version }}; then
+            echo "released=true" >> $GITHUB_OUTPUT
+          else
+            echo "released=false" >> $GITHUB_OUTPUT
+          fi
+      - name: Build Antrea image if required
+        if: ${{ steps.check-release.outputs.released == 'false' }}
+        run: |
+          ./hack/build-antrea-linux-all.sh --pull
```

Comment:
Maybe add a comment on this workflow to highlight it's only for Linux.

Reply:
added